setup.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# Distributed under the terms of the BSD 3-Clause License.
+# The full license is in the file LICENSE, distributed with this software.
+#
+# Author: Philipp Schmidt <philipp.schmidt@xfel.eu>
+# Copyright (c) 2020, European X-Ray Free-Electron Laser Facility GmbH.
+# All rights reserved.
+
+from pathlib import Path
+import re
+
+from setuptools import setup, find_packages
+
+
+with open(Path(__file__).parent / 'README.md', 'r') as f:
+    long_description = f.read()
```

Comment:
As you're using pathlib anyway, you could use the `.read_text()` method as a shortcut.